src/config/passport.js

```diff
@@ -0,0 +1,45 @@
+import passport from 'passport';
+import {
+  Strategy as GoogleStrategy
+} from 'passport-google-oauth20';
+import env from './env-config';
+import Helpers from '../utils/helpers';
+
+const FacebookStrategy = require('passport-facebook').Strategy;
+
+const {
+  passportCb
+} = Helpers;
+
```

Comment:
If you imported the helper method correctly as it is done in the codebase, lines 10-13 would not be necessary.